src/main/java/com/ita/if103java/ims/controller/UserController.java

```diff
@@ -102,6 +104,7 @@ public void updatePassword(@AuthenticationPrincipal User user,
 
     @GetMapping("/me")
     public UserDto getCurrentUser(@AuthenticationPrincipal User user) {
+        System.out.println(user);
         return mapper.toDto(user);
     }
```

Comment:
Is this method for test purposes?
@AuthenticationPrincipal User user
Shouldn't here be UserDetailsImpl class?

Reply:
Yes. It is for test purposes. For now, I am thinking about enhancing it. Just use @AuthenticationPrincipal UserDetailsIml and return User or return UserDto